src/reference-designs/tap-networking.md

```diff
@@ -0,0 +1,33 @@
+# Tanzu Application Platform Networking
+
+Tanzu Application Platform Networking diagram describes the Tanzu Application Platform most common cross clusters and external services scenarios networking in multi cluster kubernetes environment. It describes high level egress and ingress networking flows between the Tanzu Application Platform components and with the external services. 
```

Comment:
The following Tanzu Application Platform networking diagram shows the most common cross-cluster and external-services networking scenarios in a multi cluster Kubernetes environment. The diagram shows the high level egress and ingress networking flows between the Tanzu Application Platform components and the external services.